### DISCUSSION TITLE
401/403 Error when trying to access public lists with unauthed GET request

### DISCUSSION THREAD

**Discussion by benmek** [trakt/trakt-api#391] [Q&A]
Starting at around 1:00 am UTC today (2-Mar), I started getting 401 and 403 error responses when trying to access a **public** list. 

Here is an example of the `GET` call I am making: `https://api.trakt.tv/users/{user}/lists/{list}/items?limit=1000&page=1`

Anybody else suddenly experiencing this issue?

Was there a recent change that no longer allows unauthed access to public lists?

**Comment by rectifyer** [comment]
Please post a full api request and response so I can take a look with a specific example. Thanks.
